clusterloader2/pkg/measurement/common/resource_usage.go

```diff
@@ -58,6 +59,12 @@ type resourceUsageMetricMeasurement struct {
 // - start - Starts resource metrics collecting.
 // - gather - Gathers and prints current resource usage metrics.
 func (e *resourceUsageMetricMeasurement) Execute(config *measurement.Config) ([]measurement.Summary, error) {
+	cloudProvider := config.ClusterFramework.GetClusterConfig().Provider
+	if cloudProvider.Name() == provider.AutopilotName {
```

Comment:
Could you do it as cloud provider feature https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/provider/provider.go#L36:6 and not filter it by provider name, please?

Reply:
Done.